src/MediaManagerBundle/Resources/public/js/mediaManager.js

```diff
@@ -1014,7 +1014,7 @@
             var layover = self.openLayover(title, '');
 
             if (typeof height === 'undefined') {
-                height = layover.height() - layover.find('.title').height() - 50;
+                height = $('#edit-container').height() - (layover.find('.title').height() - 50);
```

Comment:
This ` - (layover.find('.title').height() - 50)` is now actually a + 24px.

Reply:
I'm not 100% sure what this calculations reason was anyway? Should we change this again?